tck/methods/key.go

```diff
@@ -0,0 +1,33 @@
+package methods
+
+import (
+	"context"
+	"github.com/hashgraph/hedera-sdk-go/v2"
+)
+
+type KeyService struct {
+}
```

Comment:
```
type AccountService struct {
	sdkService *SDKService
	hedera.AccountID
}
```
Shouldn't this be defined in similar fashion?

Reply:
The keys generation does not make requests to the network, so it does not need client.